plugins/destination/csv/client/migrate.go

```diff
@@ -0,0 +1,77 @@
+package client
+
+import (
+	"context"
+	"encoding/csv"
+	"errors"
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func isSameColumns(record []string, columns []string) bool {
+	if len(record) != len(columns) {
+		return false
+	}
+	for i, v := range record {
+		if v != columns[i] {
+			return false
+		}
+	}
+	return true
+}
+
+func (c *Client) migrate(tables schema.Tables) error {
+	for _, t := range tables {
+		filePath := path.Join(c.csvSpec.Directory, t.Name+".csv")
+		if _, err := os.Stat(filePath); err == nil {
```

Comment:
call/use as `migrateTable(t, filePath)` maybe? would solve the all the linting issues.